data/data/azure/vnet/nsg.tf

```diff
@@ -0,0 +1,69 @@
+resource "azurerm_network_security_group" "master" {
+  name                = "${var.cluster_id}-master-nsg"
+  location            = "${var.region}"
+  resource_group_name = "${var.resource_group_name}"
+
+  security_rule {
+    name                       = "AllowSSH"
+    priority                   = 100
+    direction                  = "Inbound"
+    access                     = "Allow"
+    protocol                   = "Tcp"
+    source_port_range          = "22"
+    destination_port_range     = "22"
+    source_address_prefix      = "*"
+    destination_address_prefix = "VirtualNetwork"
+  }
+
+  tags = "${merge(map(
+    "Name", "${var.cluster_id}-master-nsg",
+  ), var.tags)}"
+}
+
+resource "azurerm_subnet_network_security_group_association" "master" {
+  subnet_id                 = "${azurerm_subnet.master_subnet.id}"
+  network_security_group_id = "${azurerm_network_security_group.master.id}"
+}
+
+resource "azurerm_network_security_rule" "master_mcs" {
+  name                       = "master_mcs"
+  priority                   = 200
+  direction                  = "Inbound"
+  access                     = "Allow"
+  protocol                   = "Tcp"
+  source_port_range          = "22623"
+  destination_port_range     = "22623"
+  source_address_prefix      = "Internet"
+  destination_address_prefix = "VirtualNetwork"
+  resource_group_name        = "${var.resource_group_name}"
+
+  network_security_group_name = "${azurerm_network_security_group.master.name}"
+}
+
+resource "azurerm_network_security_rule" "master_ingress_https" {
+  name                        = "master_ingress_https"
+  priority                    = 300
+  direction                   = "Inbound"
+  access                      = "Allow"
+  protocol                    = "Tcp"
+  source_port_range           = "6443"
+  destination_port_range      = "6443"
+  source_address_prefix       = "Internet"
```

Comment:
https://github.com/openshift/installer/blob/master/data/data/aws/vpc/sg-master.tf#L44

Reply:
fixed in https://github.com/openshift/installer/pull/1454/commits/5649ea60001f1b41f842dc8675627dc423439583